src/test/java/no/difi/statistics/CategoriesQueryTest.java

```diff
@@ -0,0 +1,164 @@
+package no.difi.statistics;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.difi.statistics.elasticsearch.commands.CategoriesQuery;
+import no.difi.statistics.model.OwnerCategories;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CategoriesQueryTest {
+
+    @BeforeAll
+    static void setup() {}
```

Comment:
Disse to metodene kan slettes, siden de ikke brukes.

Reply:
Ja, de er fjernet.